### PR TITLE
docs: Add io.cozy.coachco2.settings

### PR DESCRIPTION
**Pull request by JF-Cozy**
Thanks for contributing to this documentation. If you added a doctype, please be sure that it is present 

- [x] in the `Readme.md` index page
- [x] in the `toc.yml` page
- [x] https://github.com/cozy/cozy-store/blob/master/src/locales/en.json (already done)
- [x] https://github.com/cozy/cozy-store/blob/master/src/config/permissionsIcons.json (already done)
- [x] https://github.com/cozy/cozy-store/tree/master/src/assets/icons/permissions (already done)
- [x] https://github.com/cozy/cozy-client/tree/master/packages/cozy-client/src/models/doctypes/locales (already done)
- [x] https://github.com/cozy/cozy-stack/blob/master/assets/locales/en.po (already done)
- [x] https://github.com/cozy/cozy-stack/blob/master/assets/styles/cirrus.css (already done)

### CODE REVIEW

File: docs/io.cozy.coachco2.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Cozy CoachCO2 settings doctype
+
+This doctype store information about CoachCO2 application settings. There is only one document.
```
Comment (paultranvan) [inline docs/io.cozy.coachco2.md:3]:
```suggestion
This doctype stores information about CoachCO2 application settings. There is only one document.
```

File: docs/io.cozy.coachco2.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The doctype stores the following data:
+- `account`: {Object} - Relative account, copy of `io.cozy.account` document
```
Comment (paultranvan) [inline docs/io.cozy.coachco2.md:6]:
```suggestion
- `account`: {Object} - Current account displayed, copy of the associated `io.cozy.account` document
```

File: docs/io.cozy.coachco2.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - `showAlert`: {boolean} - Either an alert is displayed requesting activation of the bike goal feature
+  - `showAlertSuccess`: {boolean} - Either an alert is displayed when the goal is reached
+  - `activated`: {boolean} - Either the feature is enabled
+  - `sendToDACC`: {boolean} - Either the user has agreed to share their data with the DACC
+  - `onboarded`: {boolean} - Either the user as completed the onboarding
+  - `onboardingStep`: {number} - The actual onboard step of the user
+  - `firstname`: {string} - The user's firstname requested in the onboarding
+  - `lastname`: {string} - The user's lastname requested in the onboarding
+  - `daysToReach`: {number} - The user's goal days to reach requested in the onboarding
+- `CO2Emission`: {Object} - Informations abount the CO2 Emission feature
```
Comment (paultranvan) [inline docs/io.cozy.coachco2.md:17]:
```suggestion
- `CO2Emission`: {Object} - Informations about the CO2 Emission feature
```

File: docs/io.cozy.coachco2.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+This doctype store information about CoachCO2 application settings. There is only one document.
+
+The doctype stores the following data:
+- `account`: {Object} - Relative account, copy of `io.cozy.account` document
+- `bikeGoal`: {Object} - Informations about the bike goal feature
+  - `showAlert`: {boolean} - Either an alert is displayed requesting activation of the bike goal feature
+  - `showAlertSuccess`: {boolean} - Either an alert is displayed when the goal is reached
+  - `activated`: {boolean} - Either the feature is enabled
+  - `sendToDACC`: {boolean} - Either the user has agreed to share their data with the DACC
+  - `onboarded`: {boolean} - Either the user as completed the onboarding
```
Comment (paultranvan) [inline docs/io.cozy.coachco2.md:12]: I don't think "either" is grammatically correct here, it's more "If" or "Whether" :)

**Review by paultranvan — APPROVED** [review]
Thanks a lot!

File: docs/io.cozy.coachco2.md (hunk 4 above)
Comment (JF-Cozy) [reply docs/io.cozy.coachco2.md:12]: Whether is fine, thanks

**Pull request merged**